Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.cpp

```diff
@@ -979,7 +1006,7 @@ RefPtr<WebCore::ShapeDetection::BarcodeDetector> WebChromeClient::createBarcodeD
 void WebChromeClient::getBarcodeDetectorSupportedFormats(CompletionHandler<void(Vector<WebCore::ShapeDetection::BarcodeFormat>&&)>&& completionHandler) const
 {
 #if ENABLE(GPU_PROCESS)
-    auto& remoteRenderingBackendProxy = m_page.ensureRemoteRenderingBackendProxy();
+    auto& remoteRenderingBackendProxy = protectedPage()->ensureRemoteRenderingBackendProxy();
```

Comment:
This doesn't seem right. Why is it safe to hold the reference from this function without protecting the page in a local variable?